litmus-portal/authentication/api/main.go

```diff
@@ -78,15 +80,27 @@ func main() {
 		log.Fatalf("failed to create collection  %s", err)
 	}
 
+	// Creating Session Collection
+	if err = utils.CreateCollection(utils.RevokedTokenCollection, db); err != nil {
+		log.Fatalf("failed to create collection  %s", err)
+	}
+
+	if err = utils.CreateTTLIndex(utils.RevokedTokenCollection, db); err != nil {
+		log.Fatalf("failed to create index  %s", err)
```

Comment:
instead of doing a fatal can you just log the error, would be great if you can change it for all collections creation